tests/vmi_configuration_test.go

```diff
@@ -105,6 +108,93 @@ var _ = Describe("Configurations", func() {
 				Expect(err).ToNot(HaveOccurred())
 			}, 300)
 		})
+
+		Context("with hugepages", func() {
```

Comment:
Can you add a functiona test which creates an unschedulable pod (or adjust one of the existing ones) and check for the schedulable condition?

Reply:
Done